web/react/stores/preference_store.jsx

```diff
@@ -122,5 +122,6 @@ class PreferenceStoreClass extends EventEmitter {
 }
 
 const PreferenceStore = new PreferenceStoreClass();
+PreferenceStore.setMaxListeners(0);
```

Comment:
I know what you mean, but it's sadly not as easy as that :-/ still thinking some more ;)